internal/pkg/create/sample.go

```diff
@@ -93,7 +93,7 @@ func getSampleReposFromGitHub(client Sampler) ([]GithubRepo, error) {
 		return []GithubRepo{}, err
 	}
 	req.Header.Set("Accept", "application/vnd.github+json")
-	req.Header.Set("X-GitHub-Api-Version", "2022-11-28")
+	req.Header.Set("X-GitHub-API-Version", "2022-11-28")
```

Comment:
😉 
```suggestion
	req.Header.Set("X-GitHub-Api-Version", "2022-11-28")
```

Reply:
Oof, scary find! 🙇🏻 Thank you! This is another drawback of branches on branches. While we fixed this in the original branch, the change still existed in this sub-branch! Yikes!

Commit 81569f1  reverts these changes.